### PR TITLE
feat : home and auth pages

### PR DESCRIPTION
**Pull request by nishasen**
Added landing page, signup and login page, also added authentication to check if the user is logged in or not

### CODE REVIEW

File: wiznotes/src/Components/AuthForms/SignupForm.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      const res = await axios.post(`/api/auth/signup`, {
+        firstName: form.firstName,
+        lastName: form.lastName,
+        email: form.email,
+        password: form.password,
+      });
+      if(res.status===201){
+        setForm(defaultForm);
+        Toast("Successfully signed up", "success")
+        //localStorage.setItem("userToken", res.data.encodedToken);
```
Comment (sadathshariff) [inline wiznotes/src/Components/AuthForms/SignupForm.jsx]: remove the commented code

**Review by sadathshariff — APPROVED** [review]
Overall the code looks good, liked the part how you are handling the form validations

File: wiznotes/src/Components/AuthForms/AuthForms.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
+.main {
+    height: 100vh;
+    margin-top: 0rem;
+}
+
+.auth-card {
+    display: flex;
+    margin-bottom: 4rem;
+    margin-top: 5rem;
+    flex-direction: column;
```
Comment (vanshitalavange) [inline wiznotes/src/Components/AuthForms/AuthForms.css:10]: suggestion: you can use utilities from your component library here

File: wiznotes/src/Components/AuthForms/LoginForm.jsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  const [showPassword, setShowPassword] = useState(false);
+  const [error, setError] = useState({
+    email: {
+      isError: false, 
+      errorMessage: "Enter a valid mail",
+    }, password: {
+      isError: false, 
+      errorMessage: "Password must contain minimum eight characters, at least one uppercase letter, one lowercase letter and one number",
+    }
+  })
```
Comment (vanshitalavange) [inline wiznotes/src/Components/AuthForms/LoginForm.jsx:32]: like how you are handling the password input

File: wiznotes/src/Components/AuthForms/LoginForm.jsx [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+                label={"Remember me"} 
+                forLabel={"rememberMe"} 
+                name={"REMEMBER_ME"} 
+                type={"checkbox"} 
+                value={"rememberMe"} 
+                labelClassName={"text-black"} />
+                <Link to="/forgot-password" className="btn-link text-black hover-red forgot-password">Forgot your password?</Link>
+            </div>
+            {error.email.isError && <div className="error-message centered"><b>{error.email.errorMessage}</b></div>}
+            {error.password.isError && <div className="error-message centered"><b>{error.password.errorMessage}</b></div>}
```
Comment (vanshitalavange) [inline wiznotes/src/Components/AuthForms/LoginForm.jsx:124]: I like how you are handling errors

File: wiznotes/src/Components/AuthForms/SignupForm.jsx (hunk 1 above)
Comment (vanshitalavange) [inline wiznotes/src/Components/AuthForms/SignupForm.jsx]: discard comments

File: wiznotes/src/Context/AuthContext/Context.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    const [response, setResponse] = useState({});
+    const userToken = localStorage.getItem("userToken")
+    
+    useEffect(() => {
+        if(userToken){
+            setUserLogin(true);
+        }
+    }, [userToken])    
+
+    const checkUserLogin = () => {
```
Comment (vanshitalavange) [inline wiznotes/src/Context/AuthContext/Context.js:20]: I like how you created a function to check the login status

**Review by vanshitalavange — APPROVED** [review]
Overall code looks very clean and structured. You have handled all edge cases really well

File: wiznotes/src/Components/AuthForms/SignupForm.jsx (hunk 1 above)
Comment (nishasen) [reply wiznotes/src/Components/AuthForms/SignupForm.jsx]: Yes removing it

File: wiznotes/src/Components/AuthForms/AuthForms.css (hunk 2 above)
Comment (nishasen) [reply wiznotes/src/Components/AuthForms/AuthForms.css:10]: Yes will spacing in utilities then add here

File: wiznotes/src/Components/AuthForms/SignupForm.jsx (hunk 1 above)
Comment (nishasen) [reply wiznotes/src/Components/AuthForms/SignupForm.jsx]: Yes will do that

**Pull request merged**